src/app/work-item/work-item.component.scss

```diff
@@ -8,93 +8,65 @@
   display: flex;
   flex: 1;
   flex-direction: column;
-
   .types-modal {
-      background-color: $color-pf-white;
-      position: absolute;
-      top: em(88);
-      left: 0;
-      right: 0;
-      bottom: 0;
-      width: 100%;
-      z-index: 11;
-      overflow-y: scroll;
-      .xl-font{
-        font-size: em(30);
+    background-color: $color-pf-black-100;
+    position: absolute;
+    top: em(88);
+    left: 0;
+    right: 0;
+    bottom: 0;
+    width: $width100;
+    z-index: 11;
+    overflow-y: scroll;
+    .container-cards-pf {
+      margin-top: 0;
+      .detail-close {
+        margin-top: em(10);
+        padding: em(11.2) em(12.9);
+        font-size: em(14);
       }
-      .wiTypeList{
-        list-style: none;
-        li{
-            height: em(180);
-            border: 1px solid $color-pf-black-200;
-            position: relative;
-            color: $color-pf-black;
-            cursor: pointer;
-            width: calc(25% - 1.667em);
-            margin: em(10);
-            float: left;
-            .wiTypeListBlk{
-              position: absolute;
-              top:50%;
-              left: 50%;
-              @include transform(translate(-50%, -50%));
-              text-align: $textCenter;
-            }
-          }
-        li:hover {
-          color:  $color-pf-blue-400;
-        }
+      .card-pf-title{
+        text-transform: capitalize;
+      }
+      .card-pf-info{
+        min-height: 4.5em;
+        @include multiline-truncate($font-size: em(14), $line-height: 1.5, $lines-to-show: 3, $trunc-bg: transparent) ;
       }
     }
-
-  .work-item-action-bar {
-    display: flex;
-    display: -webkit-flex;
-    flex-direction: inherit;
+  }
+  .wi-full-width {
+    width: $width100;
+  }
+  .toolbar-pf {
     .list-action {
-      height: em(50);
-      display: flex;
-      align-items: center;
-      background-color: $color-pf-black-300;
-      justify-content: flex-start;
-      .list-type {
-        font-weight: 700;
-        margin-left: em(10);
-      }
-      span {
-        border-color: transparent;
-        border-style: solid;
-        border-width: 1px 0;
-        padding: 1px 10px;
+      .form-group:first-child{
+        padding-left: 0;
       }
       .move-dropdown {
         a {
           cursor: pointer;
         }
-        margin-left: em(10);
-      }
-      .filter-dropdown {
-        margin-left: em(10);
-        margin-right: auto;
       }
 
       .add-detailed-wi {
-        font-size: em(20);
-        width: em(26);
-        height: em(26);
-        background: $color-pf-blue-400;
-        color: $color-pf-white;
+        width: em(17);
+        height: em(17);
```

Comment:
```css
.add-detailed-wi {
  width: em(14);
  height: em(14);
  ...
}
```
This at least matches the icon size without any side effect.

I understand the pf-toolbar is a mess, beyond your current scope (quite some custom style override has to be added to fix it) but it isn't something we can skip for long (even if we do in this patch). It eventually needs fixing.